tasks/au/geo.py

```diff
@@ -230,6 +236,7 @@ def columns(self):
         cols = OrderedDict([
             ('geom_name', geom_name),
             ('geom_id', geom_id),
+            ('external_id', external_id),
```

Comment:
Do we want to add `external_id` for all geometries? Maybe we only need to do it for one specific geometry (the exception) to avoid the duplication (and minor disk space).

Or, alternatively, just use the 11 digit code in the `geo`, and then tweak the `data` import process to calculate the 11 digit code from the 7 digit code (just a lookup in the geography table, doing partial matching).

I guess you talked about that, I just want to be up to date.

Reply:
We didn't agree on any specific solution for this. We have decided to go with this option, since it seems 7 digit codes are built from 11 digit ones (fist + six last characters):

> Or, alternatively, just use the 11 digit code in the geo, and then tweak the data import process to calculate the 11 digit code from the 7 digit code (just a lookup in the geography table, doing partial matching).

I'll let you know when it's ready for a second CR round.

Thx!